pkg/controller/plan/kubevirt.go

```diff
@@ -1265,6 +1276,14 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 							Protocol:      core.ProtocolTCP,
 						},
 					},
+					SecurityContext: &core.SecurityContext{
+						AllowPrivilegeEscalation: &allowPrivilageEscalation,
+						RunAsNonRoot:             &nonRoot,
+						RunAsUser:                &fsGroup,
```

Comment:
Why is this needed and why to set UID from GID?

Reply:
qemu is 107:107, so it's the same user and group

Reply:
Sure but then we should be explicit about it and define `qemuUser` constant.

Reply:
i did, just the naming within the function is wrong, i'll change.